sdk/storage/storage-blob/samples/typescript/basic.ts

```diff
@@ -19,7 +19,7 @@ async function main() {
   const sharedKeyCredential = new SharedKeyCredential(account, accountKey);
 
   // Use TokenCredential with OAuth token
```

Comment:
>Use [](start = 5, length = 3)

Do we need to mention credentials from `@azure/identity` here?

Reply:
Couldn't hurt, I'll update the comments to refer to it.